x-pack/plugins/maps/public/components/layer_panel/view.js

```diff
@@ -161,7 +166,7 @@ export class LayerPanel  extends React.Component {
         </EuiFlyoutBody>
 
         <EuiFlyoutFooter className="mapLayerPanel__footer">
-          <FlyoutFooter/>
+          <FlyoutFooter activateSave={this.state.styleUpdated}/>
```

Comment:
rename `activateState` to the same prop-name. It makes it easier to see how props get passed down.

Reply:
In tandem with other comment above, this would probably just end up like:

`<FlyoutFooter stateChanged={this.props.stateChanged}/>`